tests/e2e/Services/Messaging/MessagingBase.php

```diff
@@ -202,6 +202,116 @@ public function testGetProvider(array $providers)
         $this->assertEquals($providers[0]['name'], $response['body']['name']);
     }
 
+    /**
+     * @depends testUpdateProviders
+     */
+    public function testGetProviderLogs(array $providers): void
```

Comment:
For each log test, let's add a check that creating one of the related resource increases the log count

Reply:
fixed this